crates/rune/src/runtime/unit.rs

```diff
@@ -35,10 +35,21 @@ pub type DefaultStorage = ArrayUnit;
 #[cfg(rune_byte_code)]
 pub type DefaultStorage = ByteCodeUnit;
 
-/// Instructions from a single source file.
+/// Instructions and debug info from a single source file.
 #[derive(Debug, Clone, Default, Serialize, Deserialize)]
 #[serde(bound = "S: Serialize + DeserializeOwned")]
 pub struct Unit<S = DefaultStorage> {
+    /// The information needed to execute the program.
+    #[serde(flatten)]
+    logic: Logic<S>,
```

Comment:
I went with `Logic` in the end, alternatively it could be `Executable`, or `MachineCode`.

Reply:
Thanks. I'm pretty neutral on the name itself. So let's keep this!